### PR TITLE
br refactor (1): translate backup storages

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Translated [backup storages](https://github.com/pingcap/docs-cn/pull/11316/files#diff-7d443b82dce619f250b95f447258114a6c63f150a38ceb06782f5c4aa7201b1d). The document link in the docs-cn repo is https://github.com/pingcap/docs-cn/blob/master/br/backup-and-restore-storages.md

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11316, https://github.com/pingcap/docs-cn/pull/12141
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -20,228 +20,215 @@ The following services are supported:
 | Azure Blob Storage | azure, azblob | `azure://container-name/prefix/of/dest/` |
 | Write to nowhere (for benchmarking only) | noop | `noop://` |
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply br/backup-and-restore-storages.md]: Removed in 068005a

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -1,247 +1,229 @@
 ---
```
Comment (IANTHEREAL) [inline br/backup-and-restore-storages.md:1]: after removing the external storage doc, what about dumping and lightning?

Comment (shichun-0415) [reply br/backup-and-restore-storages.md:1]: Have removed description about dumpling and lightning from this file.

File: br/backup-and-restore-storages.md [hunk 3]
```diff
@@ -1,247 +1,229 @@
...
 
-> **Note:**
->
-> It is not recommended to pass in the access key and secret access key directly in the storage URL, because these keys are logged in plain text.
+**Restore snapshot data from GCS**
 
-If neither the access key and secret access key nor IAM role ARN (`role-arn`) and external ID (`external-id`) are provided, the migration tools try to infer these keys from the environment in the following order:
+```shell
+./br restore full --pd "${PD_IP}:2379" \
+--storage 'gcs://external/backup-20220915?credentials-file=${credentials-file-path}'
```
Comment (IANTHEREAL) [inline br/backup-and-restore-storages.md:98]: we should unify them `${secret_access_key}`, `${credentials-file-path}`, `${account key}`

File: br/backup-and-restore-storages.md [hunk 4]
```diff
@@ -1,247 +1,229 @@
...
+<div label="Amazon S3" value="amazon">
 
-> **Note:**
->
-> - When using Azure Blob Storage as the external storage, you should set `send-credentials-to-tikv = true` (which is set by default). Otherwise, the backup task will fail.
-> - `$AZURE_CLIENT_ID`, `$AZURE_TENANT_ID`, and `$AZURE_CLIENT_SECRET` respectively refer to the application ID `client_id`, the tenant ID `tenant_id`, and the client password `client_secret` of the Azure application. For details about how to confirm the presence of the three environment variables, or how to configure the environment variables as parameters, see [Configure environment variables](/br/backup-storage-azblob.md#configure-environment-variables).
+Before backup, configure the following privileges to access the backup directory on S3.
 
-## Command-line parameters
+- Minimum privileges for TiKV and `br` to access the backup directories during backup: `s3:ListBucket`, `s3:PutObject`, and `s3:AbortMultipartUpload`
```
Comment (IANTHEREAL) [inline br/backup-and-restore-storages.md:130]: I think we need to change `br` to BR, these permission requirements are the same for all forms of BR(br cli, tidb operator backup)

**Comment by IANTHEREAL** [comment]
Rest LGTM

File: br/backup-and-restore-storages.md (hunk 4 above)
Comment (shichun-0415) [reply br/backup-and-restore-storages.md]: Fixed in [ef1a507](https://github.com/pingcap/docs/pull/11300/commits/ef1a507558228dc0cad228751a57fddfc4b1e977)

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by IANTHEREAL** [comment]
LGTM

**Review by IANTHEREAL — APPROVED** [review]

**Pull request merged**